tools/gdb/gen_gdb_inlined_script.php

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env php
```

Comment:
Instead of introducing a new directory tools, there is already `scripts/dev`. I know that might seem annoying but new directory will duplicate places where the command-line tools and scripts should go. The tools directory does sound nice but can this somehow be synced together into a single directory?